osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -33,6 +33,10 @@ public class Slider : OsuHitObject, IHasCurve
         public double ProgressAt(double progress) => CurveObject.ProgressAt(progress);
         public int RepeatAt(double progress) => CurveObject.RepeatAt(progress);
 
+        public List<List<SampleInfo>> RepeatSamples => CurveObject.RepeatSamples;

```

Comment:
these three should probably be combined into one list (and not stored in the curve).